cvpy/image.py

```diff
@@ -234,4 +234,35 @@ def fetch_geometry_info(imdata, n=0, qry='', posCol='_position_', oriCol='_orien
     pos = struct.unpack('=%sd'%dim, example_rows[posCol][0][0:dim*8])
     ori = struct.unpack('=%sd'%(dim*dim), example_rows[oriCol][0][0:dim*dim*8])
     spa = struct.unpack('=%sd'%dim, example_rows[spaCol][0][0:dim*8])
-    return pos, ori, spa
\ No newline at end of file
+    return pos, ori, spa
+
+def get_image_array_const_component_type(image_binaries, dimensions, resolutions, ctype, n, channel_count=1):
```

Comment:
This is completely my fault, but ctype actually stands for "channel type" and not "component type". Also, now that I think about it, it might be easier to use "ctype" in the name instead of the longer version. Can you modify the function name to be "get_image_array_const_ctype"?